src/test/java/org/jabref/logic/exporter/XmpExporterTest.java

```diff
@@ -55,10 +57,23 @@ public void exportSingleEntry(@TempDirectory.TempDir Path testFolder) throws Exc
         entry.setField("author", "Alan Turing");
 
         exporter.export(databaseContext, file, encoding, Collections.singletonList(entry));
-
-        List<String> lines = Files.readAllLines(file);
-        assertEquals(15, lines.size());
-        assertEquals("<rdf:li>Alan Turing</rdf:li>", lines.get(4).trim());
+        String actual = Files.readAllLines(file).stream().collect(Collectors.joining("\n")); //we are using \n to join, so we need it in the expected string as well, \n would fail
```

Comment:
I don't understand the comment, especially the `\n would fail` part. You can also simply compare the file contents as arrays, then there is no ambiguity concerning line endings.

Reply:
The comment should have been `\r\n` would fail.
I prefer this equals variant because at least eclipse shows the differences highlighted